app/javascript/components/pages/About.js

```diff
@@ -1,10 +1,87 @@
 import React, { Component, Fragment } from 'react'
+import { Link } from 'react-router-dom'
+import { FaLinkedin, FaGithubSquare } from 'react-icons/fa'
+import team from '../../aboutData'
 
 export class About extends Component {
 	render() {
 		return (
 			<Fragment>
-				<h1>About Component</h1>
+				<div className='about-container'>
+					<div className='heading'>
+						<h1>About Us</h1>
+					</div>
+
+					<div className='about-group-content'>
+						<p>
+							Study Up was an idea implemented by four enthusiastic Web
+							Development bootcamp grads, as an easy way for kids and adults to
+							study using flashcards. Together the Developers Amanda, Alan,
+							Chris and Matt organized their talents, to create such an app to
+							give to the world. Each teammate was represented with unique
+							skills to incorporate into Study Up and together they teamed up to
+							become known as “The Console.Loggers.”
+						</p>
+
+						<p>
+							Amanda the “Tech Lead”, instituted the code specs and prioritized
+							key component functionality and usability. Along with Alan, the
+							“Design Lead”, guided the page's design and executed the UI in
+							responsiveness and styling. Chris in “Communications”, clarified
+							daily changes and tested backend structure to validate each
+							database. Along with Matt, the “Product Manager”, ensured
+							requirements were met and analyzed the Dictionary APIs specs to
+							incorporate into Study Up.
+						</p>
+						<p>
+							Every aspect of Study Up individualizes their skills and
+							formulates it into a fully customizable flashcard app. That idea
+							of creating a simple flashcard app for all is only just the
+							beginning for the ambitious team. Their experience together at
+							LEARN Academy bootcamp in San Diego, has prepared them to execute
+							Web Development properly. Now maintaining a new goal of growth in
+							software creation and education. Determined, they head in
+							different directions to aid in the growth of technology, to aid in
+							the growth of humanity and to aid in the growth of the mind
+							itself, so Study Up!
+						</p>
+					</div>
+
+					<div className='team-card-container'>
+						{team.map((person, index) => {
+							return (
+								<div className='card' key={index}>
+									<div className='person-img'>
+										<img src={person.image} alt='' />
+									</div>
```

Comment:
for accessibility purposes, it'd be good to include a description for alt attributes